engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/typer/TypeMethodReference.scala

```diff
@@ -1,22 +1,20 @@
 package pl.touk.nussknacker.engine.spel.typer
 
-import cats.data.NonEmptyList
-import org.springframework.expression.spel.ast.MethodReference
 import pl.touk.nussknacker.engine.api.process.ClassExtractionSettings
 import pl.touk.nussknacker.engine.api.typed.typing._
 import pl.touk.nussknacker.engine.definition.TypeInfos.{ClazzDefinition, MethodInfo}
 import pl.touk.nussknacker.engine.types.EspTypeUtils
 
 object TypeMethodReference {
-  def apply(methodReference: MethodReference, currentResults: List[TypingResult])(implicit settings: ClassExtractionSettings): Either[String, TypingResult] =
-    new TypeMethodReference(methodReference, currentResults).call
+  def apply(methodName: String, currentResults: List[TypingResult], params: List[TypingResult])(implicit settings: ClassExtractionSettings): Either[String, TypingResult] =
+    new TypeMethodReference(methodName, currentResults, params).call
 }
 
 // TODO:
```

Comment:
shall we remove this comment now?

Reply:
IMO we shall not remove part with arity, because we still checking only sublist of parameters when invocation is shorted than declaration. But indeed second part is to removal

Reply:
Ok, after I discovered `Map[String, MethodInfo]` now I know why was this comment. I've removed it, now we check all overloaded methods.